Exiled.API/Features/Player.cs

```diff
@@ -536,6 +536,16 @@ public bool IsIntercomMuted
             set => ReferenceHub.characterClassManager.NetworkIntercomMuted = value;
         }
 
+        /// <summary>
+        /// Gets a value indicating whether or not the player is voice chatting.
+        /// </summary>
+        public bool IsVoiceChatting => Player.GameObject.TryGetComponent(out Radio radio) && radio.isVoiceChatting;
```

Comment:
Instead of getting it every time, it would be better to get and cache the `Radio` component inside the `Player` constructor.